website/LAUNCH-PLAN.md

```
@@ -0,0 +1,824 @@
+# Allow Azure services to connect
+az postgres flexible-server firewall-rule create \
+  --resource-group aidotnet-rg \
+  --name aidotnet-db \
+  --rule-name AllowAzure \
+  --start-ip-address 0.0.0.0 \
+  --end-ip-address 0.0.0.0
+```
+
+### Step 3: Configure App Settings
+
+Replace all `<PLACEHOLDER>` values with your actual values:
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)